client/dashboard/utils/site-plan.ts

```diff
@@ -75,6 +75,10 @@ export function getJetpackProductsForSite( site: Site ) {
 }
 
 export function getSitePlanDisplayName( site: Site ) {
+	if ( site.is_wpcom_staging_site ) {
+		return __( 'Staging site' );
+	}
```

Comment:
I wonder whether we would _always_ want the staging site's plan name to appear as "Staging site". It feels sorta contextual and might belong in the specific view instead of in this utility function.
Here's probably good for now. It might not be an issue that comes up.

Reply:
I think it's fine. In a staging site, the `site.plan` is somehow the WordPress.com Free plan, so it will be misleading regardless even without this PR. 🥹